### PR TITLE
PHPFOL-6 feat : add composer.json and .gitignore

### PR DESCRIPTION
**Pull request by toutvukantabu**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "name": "jasonbenett/phpfolio",
+    "type": "library",
+    "keywords": ["phpfolio", "What The Fabrik Community", "Jason Benetti"],
+    "description": "Composer helps you declare, manage and install dependencies of PHP projects. It ensures you have the right stack everywhere.",
```
Comment (JasonBenett) [inline composer.json:5]: We need to find a description of the project.

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+    "name": "jasonbenett/phpfolio",
+    "type": "library",
+    "keywords": ["phpfolio", "What The Fabrik Community", "Jason Benetti"],
```
Comment (JasonBenett) [inline composer.json:4]: You can remove my full name from the keywords.

File: composer.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    "description": "Composer helps you declare, manage and install dependencies of PHP projects. It ensures you have the right stack everywhere.",
+    "license": "MIT",
+    "authors": [{
+            "name": "Jason Benetti",
```
Comment (JasonBenett) [inline composer.json:8]: This is not my name 😂

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+            "email": ""
```
Comment (JasonBenett) [inline composer.json:9]: jason.benedetti@live.fr

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+
+        },
+        {
+            "name": "What The Fabrik Community",
+            "homepage": "https://whatthefabrik.fr"
+
+
```
Comment (JasonBenett) [inline composer.json:16]: Pls remove useless blank lines

File: composer.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+        }
+    ],
+    "config": {
+        "sort-packages": true
+    },
+    "require": {
+        "php": " ^7.4 || ^8.0"
+    },
+    "require-dev": {
+        "php": "^7.4 || ^8.0"
```
Comment (JasonBenett) [inline composer.json:26]: No need to repeat in the require dev section what is already in the require section

File: composer.json [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+        },
+        {
+            "name": "What The Fabrik Community",
+            "homepage": "https://whatthefabrik.fr"
+
+
+        }
+    ],
+    "config": {
```
Comment (JasonBenett) [inline composer.json:19]: Let's move this to the bottom of the file. It's less important to see it first

File: composer.json [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+    },
+    "require": {
+        "php": " ^7.4 || ^8.0"
+    },
+    "require-dev": {
+        "php": "^7.4 || ^8.0"
+    },
+    "autoload": {
+        "psr-4": {
+
```
Comment (JasonBenett) [inline composer.json:30]: Nice, let's consider that our codebase will be in a "src" folder. And our project root namespace will be "JasonBenett/Phpfolio". Please update accordingly.

**Review by JasonBenett — CHANGES_REQUESTED** [review]

File: composer.json (hunk 7 above)
Comment (toutvukantabu) [reply composer.json:19]: ok

File: composer.json (hunk 4 above)
Comment (toutvukantabu) [reply composer.json:9]: ok

File: composer.json (hunk 5 above)
Comment (toutvukantabu) [reply composer.json:16]: ok

File: composer.json (hunk 6 above)
Comment (toutvukantabu) [reply composer.json:26]: ok, i 've removed "require-dev"

File: composer.json (hunk 2 above)
Comment (toutvukantabu) [reply composer.json:4]: ok

File: composer.json [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+            "name": "Jason Benedetti",
+            "email": "jason.benedetti@live.fr"
+        },
+        {
+            "name": "What The Fabrik Community",
+            "homepage": "https://whatthefabrik.fr"
+        }
+    ],
+    "require": {
+        "php": " ^7.4 || ^8.0"
```
Comment (JasonBenett) [inline composer.json:17]: "^7.4|^8.0"

File: .gitignore [hunk 10]
```diff
@@ -0,0 +1 @@
+/vendor/
```
Comment (JasonBenett) [inline .gitignore:1]: Add composer.lock too + run composer install just to be sure that it works

Comment (toutvukantabu) [reply .gitignore:1]: ok

File: composer.json (hunk 9 above)
Comment (toutvukantabu) [reply composer.json]: ok

**Comment by toutvukantabu** [comment]
It's good for me, I'm awaiting your return

**Review by JasonBenett — APPROVED** [review]

**Review by Dev-Int — APPROVED** [review]

**Pull request merged**